pkg/client/patch.go

```diff
@@ -43,9 +46,15 @@ func (s *patch) Data(obj runtime.Object) ([]byte, error) {
 	return s.data, nil
 }
 
+// RawPatch constructs a new Patch with the given PatchType and data.
+func RawPatch(patchType types.PatchType, data []byte) Patch {
+	return &patch{patchType, data}
+}
+
 // ConstantPatch constructs a new Patch with the given PatchType and data.
+// Deprecated
```

Comment:
nit: add a new line between these, and then do something like

```go
//
// Deprecated: use RawPatch instead
```

Reply:
By new line you mean:
```
// ConstantPatch constructs a new Patch with the given PatchType and data.
//
// Deprecated: use RawPatch instead
```
?

Reply:
Yeah

Reply:
(there's automated tooling that folks run that knows how to look for a line in that format)